#! /bin/sh /usr/share/dpatch/dpatch-run
## 204_manpage.diff               author unknown
##
## DP: The file help.txt is installed as slrn-help.txt in Debian.
## DP: So update the manpage.

@DPATCH@
--- slrn-0.9.8.0.orig/doc/slrn.1
+++ slrn-0.9.8.0/doc/slrn.1
@@ -208,7 +208,7 @@
 You can set this variable to a file slrn
 should read its online help from.  This is only needed when the default key
 bindings have been changed and you want the help function to reflect this.
-If unset, slrn looks for help.txt in the configuration directory.
+If unset, slrn looks for slrn-help.txt in the configuration directory.
 .TP
 .B SLRNHOME
 When interpreting filenames as relative to your home directory,
